### DISCUSSION TITLE
Mobile Apps (IOS & Android)

### DISCUSSION THREAD

**Discussion by fredygerman** [fredygerman/umeme-upo#17] [Ideas]
More than 70% of visitors to the Umeme Upo project have been from mobile, And many people are interested in notification features. This shows there is a great need for us to develop a mobile app for Umeme Upo. 

If anyone has any suggestions on this please help out.

**Comment by reubenwedson** [comment]
@fredygerman 
https://capacitorjs.com/
Same codebase across multiple platforms.

**Reply by reubenwedson** [reply]
@fredygerman another one https://solito.dev/

**Reply by fredygerman** [reply]
That will be great @reubenwedson, let's start working on this.

I will create a new feature branch for this and we will merge to main after everything works well.

**Reply by reubenwedson** [reply]
Okay keep me posted.

**Comment by fredygerman** [comment]
it will be great if we also get a redesign that will focus on being mobile-first and more responsive on all platforms. Mobile, Tablet & Desktop. 

@makindajack

**Reply by makindajack** [reply]
Working on it...
